Mods/ArchipelagoPlayer.SC2Mod/enUS.SC2Data/LocalizedData/GameHotkeys.txt

```diff
@@ -469,17 +484,9 @@ Button/Hotkey/AP_zergmissileweapons2=A
 Button/Hotkey/AP_zergmissileweapons3=A
 Button/Hotkey/AP_zergmissileweapons4=A
 Button/Hotkey/AP_zergmissileweapons5=A
-Button/Hotkey/AP_TornadoMissile=E
-Button/Hotkey/AP_WarHound=R
-Button/Hotkey/AP_StimRedirect=T
-Button/Hotkey/AP_HellbatCharge=J
-Button/Hotkey/AP_GenerateCreep=G
-Button/Hotkey/AP_StopGenerateCreep=C
-Button/Hotkey/AP_SpawnChangeling=C
-Button/Hotkey/AP_Contaminate=E
-Button/Hotkey/AP_MorphToOverseer=V
-Button/Hotkey/AP_FleetwideJump=T
-Button/Hotkey/AP_VoidRaySwarmDamageBoost=E
-Button/Hotkey/AP_CarrierHyperjump=T
-Button/Hotkey/AP_InterceptorTaldarim=I
-Button/Hotkey/AP_InterceptorPurifier=I
+Button/Hotkey/Cancel2=Escape
+Button/Hotkey/Cancel3=Escape
+Button/Hotkey/Cancel4=Escape
+Button/Hotkey/NydusCanal2=N
+Button/Hotkey/NydusCanal3=N
+Button/Hotkey/NydusCanal4=N

```

Comment:
These hotkeys probably shouldn't have been added. Sorting the other contents of *Strings.txt may be good, though it does create an extra thing to review/verify.